deploy/crds/push_v1alpha1_unifiedpushserver_crd.yaml

```diff
@@ -69,6 +69,15 @@ spec:
                 - backendSecretName
                 type: object
               type: array
+            oAuthResourceRequirements:
+              type: object
```

Comment:
We need to check if there's a way to have the nested fields written here using some `kubebuilder` magic.

Reply:
what do you mean? Would you like to verify it?

Reply:
I asked this in RHMI and OperatorFW channels and received no answer. Let's merge w/o this improvement